README.md

```diff
@@ -47,6 +47,7 @@ The second task of Radish is a CLI to accomplish a number of tasks:
 | NGINX_WORKER_PROCESSES | Number of worker processes for Nginx configuration. Default 1.|
 | RADISH_SIGNAL_FORWARD_DELAY | The delay in second from a signal is received by radish until it is sent to the child process. Default is 0 |
 | NGINX_PROXY_READ_TIMEOUT | Read timeout configuration. Default is 60 |
+| LOG_INDEX_STRATEGY | log indexing strategy is either set to `file` or `stdout`. Note: The `stdout` strategy is only available in OCP3 clusters.
```

Comment:
Kunne tenke meg at få vite hva det betyr (hvilke filer man faktisk skriver til, for eksempel) i dette output.

Reply:
Sjekk om det er enklere å forstå nå